packages/prices/utils/price.ts

```diff
@@ -137,6 +137,22 @@ const splitDecimal = (
 	};
 };
 
+const applyDecimal = (
+	afterDecimal: string,
+	decimalSeparator: string,
+	minorUnit: number
+): string => {
+	if ( afterDecimal ) {
+		return `${ decimalSeparator }${ afterDecimal }`;
+	}
+
+	if ( minorUnit > 0 ) {
+		return `${ decimalSeparator }${ '0'.repeat( minorUnit ) }`;
```

Comment:
Concerning my above note, we could use `minorUnit` minus `afterDecimal` strlen.

Reply:
I'm not sure if that's applicable here, as this part can only be reached if `afterDecimal` is empty. Therefore, `afterDecimal.strlen` should always result in `0` in this part. That's why I'm repeating the zeros based on the `Number of decimals` selected by the merchant. 

Was I missing something in this implementation?

Reply:
Ok should be good. The minor units we're converting from mitigate this.